packages/jest-types/__typechecks__/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tsconfig.json",
```

Comment:
it shouldn't extend the on in `jest-types`?

Reply:
Possible, but then where is an issue with `"rootDir": "src"`. Also `exclude`, `include` seems to be tricky. There is an error in CI. Will figure it out.